packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -100,22 +104,32 @@ final invertOversizedImages = ToggleableServiceExtensionDescription<bool>._(
   icon: const Icon(Icons.image, size: actionsIconSize),
   enabledValue: true,
   disabledValue: false,
-  enabledTooltip: 'Disable Invert Oversized Images',
-  disabledTooltip: 'Enable Invert Oversized Images',
   gaScreenName: analytics_constants.inspector,
   gaItem: analytics_constants.debugBanner,
+  tooltipContents: const ExtensionTooltipContents(
+    text:
+        'Highlight images that use more memory than necessary by turning them upside down and inverting their colors.',
+    // TODO: Correct URL
+    url:
+        'https://flutter.dev/docs/development/tools/devtools/inspector#debugging-layout-issues-visually',
+  ),
 );
 
 final debugPaint = ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.debugPaint',
-  description: 'Debug Paint',
```

Comment:
not sure if you need to update this in this PR, since they are changed in Button Redesign PR